.env.example

```
@@ -0,0 +1,17 @@
+# Stripe
+NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_test_...
+STRIPE_SECRET_KEY=sk_test_...
+
+# Cloudinary
+NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME=your_cloud_name
+
+# reCAPTCHA (contact form)
+NEXT_PUBLIC_RECAPTCHA_SITE_KEY=your_site_key
+
+# BetterAuth
+BETTER_AUTH_SECRET=your_secret_here  # Generate with: openssl rand -base64 32
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)